handlers.js

```diff
@@ -415,28 +916,132 @@
             if (! WSC.template_data) {
                 return this.renderDirectoryListing(results)
             }
+            if (results.length > 0) {
+                function DirRenderFinish() {
+                    var data = html.join('\n')
+                    data = new TextEncoder('utf-8').encode(data).buffer
+                    this.writeChunk(data)
+                    this.request.connection.write(WSC.str2ab('0\r\n\r\n'))
+                    this.finish()
+                }
+                function sendFileList() {
+                        results[w].getMetadata(function(file) {
+                            //console.log(file)
+                            var rawname = results[w].name
+                            //from https://stackoverflow.com/questions/10420352/converting-file-size-in-bytes-to-human-readable-string/10420404
+                            function humanFileSize(bytes, si=false, dp=1) {
```

Comment:
for better organization this would be better defined in like a utils.js file or something

Reply:
I think the same thing, I tried to do it and everything got a little crazy which is why I just put it here.